quadrat/sass/blocks/_navigation.scss

```diff
@@ -45,7 +45,7 @@
 						.wp-block-pages-list__item__link,
 						.wp-block-navigation-link__content {
 							padding: 0;
-							font-size: 15px;
+							font-size: var(--wp--preset--font-size--tiny);
```

Comment:
There's a font size declaration in line 31 of this file too..

Reply:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/2532bae61948e88efce92f0cb2b3f7adf7d75527 and did a sweep in the theme for other areas of font-size in https://github.com/Automattic/themes/pull/4683/commits/51266f3eea9ac012a6606ccf601ef253d6f68976